tests/helpers.py

```diff
@@ -398,168 +396,126 @@ def get_cephfs_data_pool_name():
 
 def validate_cephfilesystem(fs_name):
     """
-     Verify CephFileSystem exists at ceph and k8s
+     Verify CephFileSystem exists at Ceph and OCP
 
      Args:
         fs_name (str): The name of the Ceph FileSystem
 
      Returns:
-         bool: True if CephFileSystem is created at ceph and k8s side else
+         bool: True if cephfilesystem is created at CEPH and OCP side else
```

Comment:
Not sure, if I miss something - why the `CEPH` here is whole capitalized, when few lines above (line 399), it is `Ceph` (which is IMHO the correct form).

Reply:
will update in new commit

Reply:
Done